certora/harness/MarketplaceHarness.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.8;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {Marketplace} from "../../contracts/Marketplace.sol";
+import {MarketplaceConfig} from "../../contracts/Configuration.sol";
+
+contract MarketplaceHarness is Marketplace {
+    constructor(IERC20 token_, MarketplaceConfig memory configuration) Marketplace(token_, configuration) {}
```

Comment:
This misses the `GrothVerifier`. Also reason why it's failing.

Reply:
@0xb337r007 I just realized that your branch isn't up-to-date with latest `master` that's why it probably didn't fail for you on this one:

```
❯ g l certora..master
* 57e8cd5 (origin/master, origin/HEAD, master, chore/add-to-board-task) feat: expiry specified as duration (#99)
* 4d9320a Update .tool-versions (#94)
* a58427e Updates dist-test circuit (#96)
* 8ecc5bf readme: update todo list, we have actual proofs
* 53999c7 Provide all gas to precompiles
* 84eba26 Document that group elements are checked by precompiles
* c55b34f uint -> uint256
* 3b6f7b8 Rename _Q -> _R
* ab1b91f Return false when incorrect amount of public inputs
* bd489c7 Groth16Verifier implements its interface
* f9637f1 Rename: vkX -> combination
* a4ce10f One less addition
* df58f2d Formatting
* 5762544 Return false when public inputs are invalid
* d38e0f5 make functions private
* b676b24 Improve argument names
* 235f11a Primes are named as in EIP-197
* a4777ba Reordering, formatting
* c7687c5 Remove library
* 3840e2b Remove unnecessary if-statement
```